src/__tests__/Search.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+  const validProps = {
+    setResults: jest.fn(),
+  };
+
+  const { asFragment } = render(<Search {...validProps} />);
+
+  test("renders correctly", () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
```

Comment:
Looking over this now, I made a bit of a mistake when talking to you about this the other night... You can actually test whether or not the button has been clicked or not. As the submit function is an asynchronous one, the accompanying test also has to be:
```
test("check function fires", async () => {
  render(<Search {...validProps} />);
  const button = screen.getByText('Search');
  await fireEvent.click(button);

  expect(validProps.setResults).toHaveBeenCalled();
})
```